src/sbvr-api/sbvr-utils.ts

```diff
@@ -59,7 +59,6 @@ import {
 	statusCodeToError,
 	TranslationError,
 	UnauthorizedError,
-	AnyObject,
```

Comment:
Github won't let me comment there, but it seems we're re-exporting `./permissions` on line 46 above. Can that be removed?

Reply:
It's also a major change, but I do plan to